pandas/_libs/tslibs/c_timestamp.pyx

```diff
@@ -320,7 +320,7 @@ cdef class _Timestamp(datetime):
         cdef:
             int64_t val
             dict kwds
```

Comment:
we type these as uint_8

i don’t believe we use npy_bool anywhere and would be a consistently change